pkg/models/user.go

```diff
@@ -24,6 +24,9 @@ type User struct {
 	CreatedAt      time.Time `json:"created_at" bson:"created_at"`
 	LastLogin      time.Time `json:"last_login" bson:"last_login"`
 	EmailMarketing bool      `json:"email_marketing" bson:"email_marketing"`
+	MFA            bool      `json:"status_mfa" bson:"status_mfa"`
+	Secret         string    `json:"secret" bson:"secret"`
+	Codes          []string  `json:"codes" bson:"codes"`
```

Comment:
I think we will need a migration here to avoid parsing problems.

Reply:
a migration was added to include the necessary fields in 'user'